arbitrary-users-patch/base_images

```diff
@@ -1,18 +1,17 @@
-che-antora-2.3          docker.io/antora/antora:2.3.3
-che-cpp-rhel7           registry.access.redhat.com/devtools/llvm-toolset-rhel7
-che-dotnet-2.2          mcr.microsoft.com/dotnet/core/sdk:2.2-stretch
-che-dotnet-3.1          mcr.microsoft.com/dotnet/core/sdk:3.1.301-buster
-che-golang-1.12         golang:1.12-stretch
-che-golang-1.14         golang:1.14-stretch
-che-java11-gradle       gradle:6.6.1-jdk11
-che-java11-maven        maven:3.6.3-jdk-11
-che-java8-maven         maven:3.6.1-jdk-8
-che-nodejs10-community  node:10.16
-che-nodejs12-community  node:12.18
-che-nodejs10-ubi        registry.access.redhat.com/ubi8/nodejs-10
-che-nodejs8-centos      registry.centos.org/che-stacks/centos-nodejs
-che-php-7               quay.io/eclipse/che-php-base:7.4
-che-python-3.6          centos/python-36-centos7:1
-che-python-3.8          python:3.8.6-slim
-che-quarkus             quay.io/quarkus/centos-quarkus-maven:21.0-java11
-che-rust-1.39           rust:1.39.0-slim
+che-antora-2.3          docker.io/antora/antora@sha256:fbf31b3be4e5dd215c82f9b5b403c5e746f68bcf50ced9b7d41b6d1414aa5714
```

Comment:
do we need a script that fetches digest and update this file ? Or like a .lock file with tag/digest mapping
Because it looks like it will be really hard to maintain these digests ( like to know the original tag alias for the digest, check for updates, etc.)

Reply:
seems like we'd want something akin to what's in the related images of the che operator CSVs...
* given a list of images:tags, 
* generate a list of images@sha256:digests (using `skopeo` and `jq`), but 
* retain the source file w/ the tags so we can 
* fetch a new list of digests at intervals. 

Then we can:
* stay current but also stay locked down, and be able to 
* roll back or manually replace a tag w/ a digest if we know latest is broken and we want something older

Reply:
I don't think a script will solve the problem, the issue we had yesterday was that the one of the tags was overwritten and it broke the build. +1 for a file that shows the mappings though

Reply:
+1, new 3-column format rules.